source/Irrlicht/CMakeLists.txt

```diff
@@ -356,6 +365,7 @@ target_include_directories(Irrlicht PUBLIC
 	"$<$<CONFIG:RELEASE>:${IRRLICHT_CONF_DIR_RELEASE}>"
 )
 target_include_directories(Irrlicht PUBLIC ${IRR_ROOT_PATH}/src ${IRR_ROOT_PATH}/source/Irrlicht ${IRR_ROOT_PATH}/src/3rdparty) # this should be PRIVATE, but things from /src (or /source) are sometimes included in things in /include and so examples have to put source dirs into theirs Include Path
+target_include_directories(Irrlicht PRIVATE ${CMAKE_BINARY_DIR}/include/irr/builtin)
```

Comment:
you don't need this line at all

Reply:
it's needed for `#include "builtinResources.h"`

Reply:
should have been `#include "irr/builtin/builtinResources.h"`

Reply:
what does it look like now?